Source/UnitTests/Core/DSP/DSPTestText.cpp

```diff
@@ -1,4 +1,10 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
 
+#include "DSPTestText.h"
+
+const char* s_dsp_test_text = R"(
```

Comment:
This should be a `const char s_dsp_test_text[8433]` array or another `const` should be added after the `char*` (i.e. `const char* const`). A compiler can't assume this is a fixed pointer to a string otherwise (most of the time), since it's still mutable.